### PR TITLE
Make `git-touch` behave like `touch` when passed multiple files

### PR DESCRIPTION
**Pull request by al-the-x**
Re #467:

* [x] use `$@` instead of `$#`
* [x] test that arguments have been passed (`$#`)
* [ ] maybe print usage message?
* [ ] test revisions

### CODE REVIEW

**Comment by al-the-x** [comment]
I haven't tested this yet, but I have metering a commit so that I could open a PR as a reminder.

**Comment by hemanth** [comment]
Let us know once it's good to go.

**Comment by qw3rtman** [comment]
@al-the-x, is this in progress?

**Comment by al-the-x** [comment]
Ready for review @qw3rtman. Please provide feedback.

File: bin/git-touch [hunk 1]
```diff
@@ -1,8 +1,13 @@
 #!/usr/bin/env bash
 
-filename="$*"
+USAGE(){
```
Comment (qw3rtman) [inline bin/git-touch]: Why capitalized?

Comment (qw3rtman) [inline bin/git-touch]: No need to write another function.

File: bin/git-touch [hunk 2]
```diff
@@ -1,8 +1,13 @@
...
+  echo "git touch <filename> [ <filename> . . .]" 1>&2
+  exit 1
+}
 
-test -z "$filename" && echo "filename required" 1>&2 && exit 1
+test $# -lt 1 && USAGE
```
Comment (qw3rtman) [inline bin/git-touch]:
```shell
test $# -lt 1 && echo "usage: git touch <filename> [<filename> ...]" 1>&2 && exit 1
```

**Comment by qw3rtman** [comment]
Everything looks good, except you don't need to write another function to display usage (it's only used once). See my line notes.

**Comment by qw3rtman** [comment]
Looks great, thanks! :+1:

**Pull request merged**